synthea-hiv/uploader/fhir_client_test.py

```diff
@@ -39,7 +39,7 @@ def setUp(self):
     super().setUp()
     mock_session = mock.MagicMock()
     mock_session.post.side_effect = mock_requests_post
-    mock.patch('fhir_client._setup_session', return_value=mock_session).start()
+    mock.patch('requests.Session', return_value=mock_session).start()
```

Comment:
If you want to keep this, it is fine, but I think this is not really needed for this PR.

Reply:
removed